tests/Filter/EmptyFieldFilterTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DoctrineORMAdminBundle\Tests\Filter;
+
+use PHPUnit\Framework\TestCase;
+use Sonata\DoctrineORMAdminBundle\Filter\EmptyFieldFilter;
+use Sonata\Form\Type\BooleanType;
+
+final class EmptyFieldFilterTest extends TestCase
+{
+    public function testRenderSettings(): void
```

Comment:
I think more tests could be great, at least two more tests, testing the query for the null case and the not null case.

Reply:
Class is now 100% covered